aframe/pipelines/sandbox/configs/base.cfg

```diff
@@ -67,6 +68,7 @@ reference_frequency = &::luigi_base::reference_frequency
 waveform_approximant = &::luigi_base::waveform_approximant
 coalescence_time = &::luigi_base::coalescence_time
 prior = &::luigi_base::prior
+request_memory = 64678 mb
```

Comment:
Im pretty sure that would work, but the condor memory syntax is so finicky that Im worried it would break things haha so will leave that to a future pr

Reply:
Hah, I see, that's good to know